tools/restatectl/src/commands/config/set.rs

```diff
@@ -75,11 +75,11 @@ async fn config_set(connection: &ConnectionInfo, set_opts: &ConfigSetOpts) -> an
 
     set_opts.log_provider.inspect(|provider| {
         match provider {
-            ProviderKind::InMemory | ProviderKind::Local => {
-                c_warn!("You are about to reconfigure your cluster with a Bifrost provider that only supports a single node cluster.");
-            }
             ProviderKind::Replicated => {
                 // nothing to do
+            },
+            _ => {
```

Comment:
Personally, I am in favor of spelling variants out. The reason is that this way we'll get notified about potentially required changes if we add a `ProviderKind` variant.